frontend/src/v5/ui/components/shared/userMenu/editProfileModal/editProfilePersonalTab/editProfilePersonalTab.component.tsx

```diff
@@ -45,7 +47,7 @@ type EditProfilePersonalTabProps = {
 	alreadyExistingEmails: string[];
 	setAlreadyExistingEmails: (emails: string[]) => void;
 	unexpectedError: any,
-	onClickClose: () => void,
+	onClickClose?: () => void,
 };
 
 export const EditProfilePersonalTab = ({
```

Comment:
My guess is that the email field should be read only here because you cant change it?

Reply:
This was addressed here https://github.com/3drepo/3drepo.io/pull/5456